test/app.test.js

```diff
@@ -0,0 +1,74 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { WebGLRenderer, PerspectiveCamera, Vector3 } from 'three';
+import { SeedScene } from 'scenes';
+import { CSS3DRenderer } from 'three/examples/jsm/renderers/CSS3DRenderer.js';
+import { Object3D } from 'three';
+import * as handlers from '../src/js/handlers.js';
+import { initializeApp, resizeHandler } from '../src/app.js';
```

Comment:
Good job getting the needed imports to set a good foundation of testing tools that can be used to further build out the testing suite later.